pycqed/tests/test_timedomain_analysis.py

```diff
@@ -24,18 +58,18 @@ def test_Rabi_analysis(self):
                 self.assertGreaterEqual(rabi_amp, amp_low)
                 self.assertGreaterEqual(amp_high, rabi_amp)
 
-        a = ma.Rabi_Analysis(timestamp='20170607_160504')
+        a = ma.Rabi_Analysis_original(timestamp='20170607_160504')
```

Comment:
I very much disapprove of deprecating a method in this manner. Existing tests should not be modified! 

The reason is that we rely on them to ensure other things do not break. 
If you do want to reimplement a method (like the Rabi analysis) for which there can be good reason, it should be made such that the original tests still pass.